test/caesar_test.rb

```diff
@@ -1,2 +1,19 @@
-# YOUR TESTS GOES HERE
-# I PITY THE FOOL WHO DOESNT WRITE TESTS
+
+require 'minitest/autorun'
+require 'minitest/pride'
+require './lib/caesar'
+require 'pry'
+
+class Caesar < Minitest::Test
+  def test_Caesar_exists
+    cipher = Caesar.new
+
+    assert_instance_of Caesar, cipher
+  end
+
+  def encrypt("the quick brown fox jumps over the lazy dog")
```

Comment:
This should be a test name, i.e. test_it_encrypts. Anything not named test_<test_name> will not be run